src/database/models.py

```diff
@@ -1609,18 +1612,18 @@ def info(self):
 
     def simple_json(self):
         data = model_to_dict(
-            self, exclude=["tags", "image", "community", "invited_communities"]
+            self, exclude=["tags", "image", "community", "invited_communities", "user"]
         )
-        data["start_date_and_time"] = self.start_date_and_time
-        data["end_date_and_time"] = self.end_date_and_time
+        #data["start_date_and_time"] = self.start_date_and_time
```

Comment:
Start date/time and end date/time for events should be included in simple_json, I think.

Reply:
I agree, but they are already included by virtue of the model_to_dict call right above it.

Reply:
Okay, I was wondering about that. Sounds good then.

Reply:
Removed those commented lines